images/installer/Dockerfile

```diff
@@ -12,23 +12,54 @@ LABEL name="openshift/origin-ansible" \
       io.k8s.display-name="openshift-ansible" \
       io.k8s.description="A containerized openshift-ansible image to let you run playbooks to install, upgrade, maintain and check an OpenShift cluster" \
       io.openshift.expose-services="" \
-      io.openshift.tags="openshift,install,upgrade,ansible"
+      io.openshift.tags="openshift,install,upgrade,ansible" \
+      vcs-url="https://github.com/openshift/openshift-ansible" \
+      vcs-type="git" \
+      version="alpha"
```

Comment:
(If we really need a `version` label, we could have our image build script set the right version on build)